bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/weather.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="netatmo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-group-type id="temperature-extended">
+		<label>Temperature</label>
+		<channels>
+			<channel id="value" typeId="system.indoor-temperature"/>
+			<channel id="min-today" typeId="min-temp"/>
+			<channel id="max-today" typeId="max-temp"/>
+			<channel id="min-time" typeId="timestamp-advanced">
+				<label>Today Min Timestamp</label>
+				<description>Moment when temperature was measured at its minimum today.</description>
+			</channel>
+			<channel id="max-time" typeId="timestamp-advanced">
+				<label>Today Max Timestamp</label>
+				<description>Moment when temperature was measured at its maximum today.</description>
+			</channel>
+			<channel id="trend" typeId="trend">
+				<label>Temperature Trend</label>
+			</channel>
+			<channel id="heat-index" typeId="heat-index"/>
+			<channel id="dewpoint" typeId="dewpoint"/>
+			<channel id="dewpoint-depression" typeId="dewpoint-depression"/>
```

Comment:
This was added quite late in commit 6380db141f4862ea891ef0e42ee5a6a3a9253bde. Was this intended? I'm not sure, but I think previously this was being added dynamically.

Reply:
Sorry @jlaur , I think I do not understand your point here.

Reply:
Sorry, I now realize that they had to be duplicated because of the separation of temperature channel groups. I mixed it up with https://github.com/openhab/openhab-addons/pull/10831#issuecomment-949507979.